### DISCUSSION TITLE
是否考虑加入习惯记录的功能？

### DISCUSSION THREAD

**Discussion by yanyu2015** [Planshit/Tai#187] [Ideas]
- 首先非常感谢开发者的这款软件，虽然同类软件也有不少，但实际使用起来都没有这款这么便捷易用，反馈效果也很一般。然后试用了3天，对我真的有帮助，比如让我认识到每天哪怕长时间待在工作岗位，但要想让工作时长达到11个小时依然是一件非常困难的事情（哪怕把一些不容易分类的软件的使用时间也算上（比如使用浏览器查资料/摸鱼））。这个软件相比于番茄钟之类的优势在于，番茄钟可能会经常忘记，但是该软件没有这种烦恼，虽然分类不会有番茄钟准确，但记录的情况也更真实。
- 其次，我想**请问下有没有增加类似[uhabits](https://github.com/iSoron/uhabits)的功能的计划**？不=用太重，能点击后记录并可视化即可。
- 最后，反馈下一个小问题，在右键对软件进行分类选择的时候，分类的子菜单列表总是难以点击（鼠标滑动过去后窗口会消失）

**Comment by noberumotto** [comment]
非常感谢能得到你的认可
我需要花时间去了解一下[uhabits](https://github.com/iSoron/uhabits)的功能才能决定，还要看一下用户的需求量，如果需求量大会列入主程序中，反之可以在未来以插件的形式提供，现在的开发计划是修复当前严重的bug后重写一遍软件，在重写完成之前暂停加入新的功能了。

**Comment by yanyu2015** [comment]
感谢，期待后面出个roadmap
